foxtrot-core/src/test/java/com/flipkart/foxtrot/core/querystore/impl/ElasticsearchQueryStoreTest.java

```diff
@@ -95,7 +94,9 @@ public void testSaveSingleRawKeyVersion1() throws Exception {
                 .prepareGet(ElasticsearchUtils.getCurrentIndex(TestUtils.TEST_TABLE_NAME, originalDocument.getTimestamp()),
                         ElasticsearchUtils.DOCUMENT_TYPE_NAME,
                         originalDocument.getId())
-                .setFields("_timestamp").execute().actionGet();
+                .setStoredFields("_timestamp")
```

Comment:
[IMPORTANT] Check the sanity of these UTs. 
We need to check if there's any regression on the _timestamp field being removed.